pycytominer/cyto_utils/output.py

```diff
@@ -74,19 +78,29 @@ def output(
         float_format=None,
     )
     """
-    # Make sure the compression method is supported
-    compression_options = set_compression_method(compression=compression_options)
-
-    df.to_csv(
-        path_or_buf=output_filename,
-        sep=sep,
-        index=False,
-        float_format=float_format,
-        compression=compression_options,
-    )
+
+    if output_type == "csv":
+
+        compression_options = set_compression_method(compression=compression_options)
+
+        df.to_csv(
+            path_or_buf=output_filename,
+            sep=sep,
+            index=False,
+            float_format=float_format,
+            compression=compression_options,
+        )
+
+    elif output_type == "parquet":
+
+        # note: compression options will be validated against pd.DataFrame.to_parquet options
+        # raising errors and tested through Pandas, PyArrow, etc. as necessary.
+        df.to_parquet(path=output_filename, compression=compression_options)
```

Comment:
Does `pd.DataFrame.to_parquet()` take the same compression options as `pd.DataFrame.to_csv()`?

I'm ok with being very opinionated about parquet compression. i.e. maybe it is best to always use one specifically (or never use one).

Reply:
Pandas `to_parquet()` vs `to_csv()` compression options are pretty different, `gzip` I think being the only overlapping option. Would you like to use the current Pandas default for `to_parquet()` compression here: `snappy`? I believe there are slight performance implications (read and write speeds) depending on what's used.

Reply:
yes, for now, let's use default pandas `snappy` compression.

I think that updating `compression_options` for `to_parquet()` might be slightly out of scope.

Reply:
Understood, thank you. I'll update this with an additional commit.